testing/gradle/APKBUILD

```diff
@@ -0,0 +1,54 @@
+# Contributor: Adrian Siekierka <kontakt@asie.pl>
+# Maintainer: Adrian Siekierka <kontakt@asie.pl>
+pkgname=gradle
+pkgver=3.3
+pkgrel=0
+pkgdesc="A Java-based build system"
+url="http://gradle.org/"
+arch="noarch"
+license="ASL-2.0"
+depends="openjdk8-jre-base"
+makedepends=""
+subpackages="$pkgname-doc"
+source="http://services.gradle.org/distributions/$pkgname-$pkgver-all.zip
+        gradle"
+builddir="$srcdir/$pkgname-$pkgver"
+pkghome="/usr/share/java/$pkgname"
+
+prepare() {
```

Comment:
When you define custom `prepare` function, please always invoke `default_prepare || return 1`, usually as the first command in the function.